### ISSUE TITLE
Gitbook

### ISSUE THREAD

**Issue by Larpoux** [dooboolab/flutter_sound#533]
V 6.4.3 uses gitbook for the documentation.

This is better than before.
But I am not completely satisfied by gitbook : 

- The license is ambiguous. Flutter Sound will never use any tool not free. Gitbook is (was) released under an Apache license. This is a free license. But if you look to the Gitbook repo, you will notice that this project is deprecated and no more supported, in favor of their own project. But their own project seems not to be free anymore. This is what happens for Projects which does not have a Copyleft clause (GPL 😄 ).

- There is bugs when we want to link one of the doc page to an HTML page (dartdoc, for ex.). We cannot use a relative link. This means that we must use absolute URL and so, the link of one branch will always link to the master branch. This is not OK.

- Hard absolute links open a new tab in Firefox/Chrome instead of using the current tab. I do not like that

- The menu on the left panel cannot be collapsed/expanded. I miss this feature

I do not have enough time to spend on finding another tool which would be **FREE** and better. 
If someone has an idea what tool we can use ...

**Comment by gedw99** [comment]
Hugo is used a lot for docs and has some great templates.

google docsy has become a defacto one.

https://github.com/google/docsy

**Comment by Larpoux** [comment]
Hi @gedw99 .

I spent some times (in fact too much time) to investigate what tool can be used for Flutter Sound.
Actually the situation is really bad : 
- Gitbook on github has not been maintained for 3 years. This free project (under Apache license) is now deprecated. This free version is buggy, and gives Security alert by Github. This is not acceptable.
- The current Gitbook project is developed by a little French Company but is not free anymore. Not good for Flutter Sound.

My actual choice is [Jekyll](https://idratherbewriting.com/documentation-theme-jekyll/index.html).

This is the only tool I found which is compatible with the documentation produced by `dartdoc`.
I really wanted that the user can go smoothly from Flutter Sound documentation to the HTML pages produced by dartdoc, without opening an external page in a new Browser tab. And then, the user can return to the Flutter Sound documentation with the Browser Back Button.
I wanted also that the Top Bar menu of Flutter Sound documentation be shown in the Dartdoc pages.
I wanted also that the side bar menu of Flutter Sound documentation be shown in the Dartdoc pages, with the possibility to remove dynamicaly this bar by the user if he has a too narrow screen.

Jekyll is the only tool I found OK with this. it is not perfect. Far from that. But with much work for me to fix all the problems of Dartdoc it will be acceptable, I think.

I hope to release my work in a few days.

Of course, if you feel that it is a bad choice, tell us.
Thank you for your input. I like when Flutter Sound users contribute.
